packages/access-api/wrangler.toml

```diff
@@ -23,20 +17,22 @@ preview_id = "62e57652625c44a3b1fef2f840ffc882"
 [[d1_databases]]
 binding = "__D1_BETA__"
 database_name = "spaces-dev"
-database_id = "7c676e0c-b9e7-4711-97c8-7b1c8eb229ae"
+database_id = "e3331d3b-d8dd-48b0-ab80-2dba93c2c256"
```

Comment:
new db because the old one is quite out of date, has no important data, and got hung up on migrations. No one has been using `dev` afaik so I think this should be safe.